### ISSUE TITLE
Year Info - gone? [BUG]

### ISSUE THREAD

**Issue by Gonreis** [kabouzeid/phonograph-issue-tracker#158]
Some albums don't show year's info on each artist page but when i open in-built tag editor that info is there.

**Comment by louissegal** [comment]
Is this with FLAC? Phonograph doesn't support FLAC years despite showing them in tag editor.

**Comment by Gonreis** [comment]
No. I only have mp3. Instead of showing the years, there is a "0" on some albums.

**Comment by rafaglez** [comment]
This is also happening to me
